intents/test_all.sh

```diff
@@ -91,6 +91,14 @@ else
   list_logs+=("oversights/test_simpsons_paradox passed")
 fi
 
+if ! python3 oversights/test_top_down_error.py;
+then
+  err "oversights/test_top_down_error failed"
+  list_logs+=("oversights/test_top_down_error failed")
+else
+  list_logs+=("oversights/test_top_down_error passed")
+fi
+
```

Comment:
Some of the tests are failing - 
1. Debiasing sentence was only updated in the code & not in assertions
2. Merge conflicts in constants.py

Reply:
Resolved this already